src/styles/index.css

```diff
@@ -0,0 +1,393 @@
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(17)::before {
+  content: '17';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(18)::before {
+  content: '18';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(19)::before {
+  content: '19';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(20)::before {
+  content: '20';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(21)::before {
+  content: '21';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(22)::before {
+  content: '22';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(23)::before {
+  content: '23';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(24)::before {
+  content: '24';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(25)::before {
+  content: '25';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(26)::before {
+  content: '26';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(27)::before {
+  content: '27';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(28)::before {
+  content: '28';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(29)::before {
+  content: '29';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(30)::before {
+  content: '30';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.calendar__day:nth-child(31)::before {
+  content: '31';
+  display: inline-block;
+  font-size: 30px;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
The styles for the calendar days are repetitive. Consider using a loop or a more dynamic approach to generate these styles if possible, or use a single class with a data attribute to set the content dynamically.